internal/templating/engine_test.go

```diff
@@ -7,7 +7,8 @@ import (
 )
 
 func TestEngineAlternateSeparator(t *testing.T) {
-	defaultTemplate, _ := NewDefaultTemplateWithPattern("topic*")
+	defaultTemplate, err := NewDefaultTemplateWithPattern("topic*")
+	require.NoError(t, err)
```

Comment:
This change causes an error now, stating `no measurement specified for template. "topic*"`. Should I just ignore this error, or is this a bug that this uncovered?

Reply:
So it seems it goes and grabs a default template pattern, then creates a new engine  with various tempaltes. So that initial template of `topic*` doesn't really get used. I'd suggest we update the test to say:

```diff
- defaultTemplate, _ := NewDefaultTemplateWithPattern("topic*")
+ defaultTemplate, err := NewDefaultTemplateWithPattern("measurement*")
+ require.NoError(t, err)
```

That should allow us to check for errors, and allow the test to continue to work as expected.

Let me know!